applications/surgical_scene_recon/training/scene/endo_loader.py

```
@@ -0,0 +1,576 @@
+        else:
+            raise ValueError(f"{self.mode} has not been implemented.")
+        self.masks_paths = get_file_paths("masks")
+
+        assert len(self.image_paths) == poses.shape[0], "the number of images should equal to the number of poses"
+        assert len(self.depth_paths) == poses.shape[0], "the number of depth images should equal to number of poses"
+        assert len(self.masks_paths) == poses.shape[0], "the number of masks should equal to the number of poses"
+        
+    def format_infos(self, split):
+        cameras = []
+        
+        if split == 'train': idxs = self.train_idxs
```

Ruff (0.14.5)

Multiple statements on one line (colon)

(E701)